src/main/webapp/content/scss/global.scss

```diff
@@ -137,12 +137,6 @@ Generic styles
 
 /* other generic styles */
 
-.jh-card {
```

Comment:
I think if you remove this css rule because it isn't needed anymore, you can also remove the corresponding class from the html in main.component.html.

Reply:
Ah well, I thought this was just extending an existing class, but apparently not. Fixed, thanks!